src/modals/updateVersionModal/updateVersionModal.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { Modal, Form, Input } from 'antd';
+import { updateVersion } from '../../services';
+
+import './styles.scss';
+
+export const UpdateVersionModal = ({ id, isOpen, onCancel }) => {
+  const [imageLinkValue, setImageLinkValue] = useState('');
+  const [versionNameValue, setVersionNameValue] = useState('');
+  const [informationValue, setInformationValue] = useState('');
+  const [warning, setWarning] = useState(false);
+
+  const handleUpdate = () => {
+    if (id && imageLinkValue && versionNameValue && informationValue) {
+      updateVersion(id, imageLinkValue, versionNameValue, informationValue);
+    } else {
+      setWarning(true);
+    }
+  };
```

Comment:
Instead of this logic you can use Antd's Forms: The validation and submit is done internally and you just need to set the rules there and not write all of this logic